src/reanimated2/hook/useAnimatedSensor.ts

```diff
@@ -52,18 +124,42 @@ export function useAnimatedSensor(
     isAvailable: false,
     config: {
       interval: 0,
+      adjustToInterfaceOrientation: false,
+      iosReferenceFrame: IOSReferenceFrame.XArbitraryCorrectedZVertical,
     },
+    interfaceOrientation: makeMutable<InterfaceOrientation>(0),
   });
 
   useEffect(() => {
-    ref.current.config = { interval: 'auto', ...userConfig };
+    ref.current.config = {
+      interval: 'auto',
+      adjustToInterfaceOrientation: false,
+      iosReferenceFrame: IOSReferenceFrame.XArbitraryCorrectedZVertical,
```

Comment:
Can we also introduce option `'best'` which uses the best possible sensor on iPhone/iPod?

Reply:
Which one is "best"? :D https://developer.apple.com/documentation/coremotion/cmattitudereferenceframe?language=objc
The first one works on iPod, the rest not, but give better results.

Reply:
Our aim here is to provide the best possible behavior out-of-the-box. React Native developers just want a simple hook to access sensor measurements without deep-diving into CoreMotion documentation. I see that the default value here is `XArbitraryCorrectedZVertical`. The question is, is it really the best possible setting across all kinds of devices? I think you've mentioned that a sensor that works the best on iPhone is not available on iPod or something similar. The idea here is to add another enum value `IOSReferenceFrame.BestAvailable` that represents sensor A for iPods and sensor B for iPhones on the native side.